.buildbot.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/sh
+
+set -e
+
+export CARGO_HOME="`pwd`/.cargo"
+export RUSTUP_HOME="`pwd`/.rustup"
+
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup.sh
+sh rustup.sh --default-host x86_64-unknown-linux-gnu --default-toolchain stable -y --no-modify-path
+
+export PATH=`pwd`/.cargo/bin/:$PATH
+
+cargo fmt --all -- --check
+cargo test
```

Comment:
Did you want to test release mode too?

Reply:
Good point! Fixed in https://github.com/softdevteam/packedvec/pull/37/commits/0cf18e15516215107b70aaad49ded8a4795b798f.